nss-day-cohort-48/rare-client-orchesta-outlaws
# Editing and Deleting Categories +++ Creating Reactions

## Files Modified
`CategoryList.js` and `CategoryProvider.js` +++ `ReactionBox.js` and `ReactionProvider.js`, plus `NavBar.js` and `ApplicationViews.js`

## Files Created
`/reaction/ReactionBox.js` and `reaction/ReactionProvider.js`

## Changes Made
Fixed functionality of editing and deleting categories;
Added "Reaction Manager" to view and add potential reactions to posts

To test: 
1. Clone repo (al-categories) and check that, once logged in, you can DELETE and EDIT categories (pending server-side PR from 'al-testbranch' [branch now merge into main in server repo])
2. Go to "Reaction Manager" and test that you can POST a new reaction (sample url: https://upload.wikimedia.org/wikipedia/commons/thumb/2/23/Facebook_Like_button.svg/1024px-Facebook_Like_button.svg.png)

> Thank you again to Val for helping me solve the issue with the constant re-rendering of the modal box upon text input
